src/app/components/LiveMediaStream/index.tsx

```diff
@@ -0,0 +1,129 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { memo, useContext, useState } from 'react';
+import Text from '#app/components/Text';
+import { MediaCollection } from '#app/components/MediaLoader/types';
+import MediaLoader from '#app/components/MediaLoader';
+import filterForBlockType from '#app/lib/utilities/blockHandlers';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import mediaIcons from '#psammead/psammead-assets/src/svgs/mediaIcons';
+import styles from './index.styles';
+import WARNING_LEVELS from '../MediaLoader/configs/warningLevels';
+
+type WarningItem = {
+  // eslint-disable-next-line camelcase
+  warning_code: string;
+  // eslint-disable-next-line camelcase
+  short_description: string;
+};
+
+type Props = { mediaCollection: MediaCollection[] | null };
+
+const DEFAULT_WATCH__NOW = 'Watch Now';
+
+const MemoizedMediaPlayer = memo(MediaLoader);
+
+const LiveMediaStream = ({ mediaCollection }: Props) => {
+  const { translations } = useContext(ServiceContext);
```

Comment:
Could be worth adding an `isLite` check here to return `null` if its Lite. Just so we don't show a button to users that they can't interact with.